### ISSUE TITLE
At what level should the settings be persisted?

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/create-app#98]
It appears that all "settings" (org, language, font sizes, book/chapter/verse) are persisted at the computer level. Using a different username or even a different browser does not reset the settings. Is that the desired behavior?

**Comment by birchamp** [comment]
@cckozie you mean a different door43 username, correct?

**Comment by cckozie** [comment]
@birchamp - Yes, I was referring to DCS username.

**Comment by PhotoNomad0** [comment]
@cckozie @elsylambert These changes are in build: v0.9.0 build 736dc08

**Comment by elsylambert** [comment]
v0.9.0 build 736dc08
Tested with two user accounts. Settings are persisted per user account as expected.

**Comment by cckozie** [comment]
This partially works correctly in v0.9.0 build f601ed4
1. I cleared all local storage
2. I logged in as tc01 choosing test_org and English
3. I changed the scripture card versions as shown here:
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/28c499a9-c1b9-45b2-9384-db17a418802c)

3. I logged out and logged in as tcc001 choosing test_org and Russian
4. The project opened as expected (I'm assuming there is no Russian Matthew text)
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/9552b120-cfe7-4bee-9c28-352d836b54ad)

5. I logged out and logged in again as tc01
6. The project opened with theses problems:
 - The book is set to Ruth instead of Matthew
 - The first 2 scripture cards have the correct title but wrong content
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/58b7d91a-f4da-4ba6-8d23-487173fec0e8)

7. I logged out and logged in again as tcc001
8. Now the version dropdown list has some unexpected options
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/3d75b2d1-3559-4b6f-92c4-96dd5d29913e)

**Comment by elsylambert** [comment]
Good catch Koz, I am seeing differently than yours. The settings are persistent on user accounts but resources dropdown gets some additional options: two screenshots from two different scenarios :

![Screen Shot 2021-04-27 at 12.18.54 AM.png](https://images.zenhubusercontent.com/5d02efb2f425650e637abe10/9dd87653-c21f-43a0-903c-cb1cc4051d61)![Screen Shot 2021-04-27 at 12.00.45 AM.png](https://images.zenhubusercontent.com/5d02efb2f425650e637abe10/8d7b0fa0-5e4d-4f70-86b5-d8eacef5a281)

**Comment by PhotoNomad0** [comment]
@cckozie Yes, you can see the list of translated books for ru by just clicking on the down arrow next to the book ID.  I'm also seeing the problems you listed.

**Comment by PhotoNomad0** [comment]
@cckozie @elsylambert Fixes are in v0.9.0 build 1351b36

**Comment by cckozie** [comment]
v0.9.0 build 008141c
The problems described above have been fixed, but I found a couple more.
The column checkbox states and the markdown view switch state are not persisted by user. Actually, they are not persisted at all.

**Comment by elsylambert** [comment]
Works as expected in v0.9.0 build 008141c.

**Comment by cckozie** [comment]
Tested in Test Persistence by User
